lib/rules/yo-version.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+var message = require('../message');
+var NpmInfo = require('npm-info');
+var npm = new NpmInfo();
+var binVersionCheck = require('bin-version-check');
+
+exports.description = 'yo version';
+
+var errors = exports.errors = {
+  yoNpm: function () {
+    return message.get('yo-version-unable-to-fetch', {});
```

Comment:
Ah, that does look like a much better alternative.  `npm-info` is a lot more slow.